### PR TITLE
AAkrit: Audio Trimmer Module

### PR DESCRIPTION
**Pull request by aakritsubedi**
- Audio Trimmer Module [https://github.com/aakritsubedi/audioTrimmer](https://github.com/aakritsubedi/audioTrimmer) 
  - using pydub library
  - `audioTrimmer('fileName',startMin,startSec,endMin,endSec)`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,4 @@
-# musicrs
\ No newline at end of file
+# musicrs
+
+### Audio Trimmer
```
Comment (Basanta78) [inline README.md]: Lets keep it in References heading since your module is the one that we are using

File: musicrs/util/audioTrimmer.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+from pydub import AudioSegment
+import sys
+sys.path.append('/usr/bin/ffmpeg')
+
+def audioTrimmer(file_name,startMin,startSec,endMin,endSec):
+  files_path = 'mp3/'
+
+  # Time to miliseconds
+  startTime = startMin*60*1000+startSec*1000
+  endTime = endMin*60*1000+endSec*1000
```
Comment (ShishirBhandari) [inline musicrs/util/audioTrimmer.py]: Do not use hardcoded values. Use variables.

File: musicrs/util/audioTrimmer.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+  # Opening file and extracting segment
+  song = AudioSegment.from_mp3( files_path+file_name+'.mp3' )
+  extract = song[startTime:endTime]
+
+  # Saving
+  extract.export( files_path+file_name+'-extract.mp3', format="mp3")
```
Comment (ShishirBhandari) [inline musicrs/util/audioTrimmer.py]: You can use logs to show information, I think there is Logger component.

File: musicrs/util/audioTrimmer.py [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+def audioTrimmer(file_name,startMin,startSec,endMin,endSec):
+  files_path = 'mp3/'
+
+  # Time to miliseconds
+  startTime = startMin*60*1000+startSec*1000
+  endTime = endMin*60*1000+endSec*1000
+
+  # Opening file and extracting segment
+  song = AudioSegment.from_mp3( files_path+file_name+'.mp3' )
```
Comment (Basanta78) [inline musicrs/util/audioTrimmer.py]: does it only work for mp3?

File: musicrs/util/audioTrimmer.py [hunk 5]
```diff
@@ -0,0 +1,17 @@
+from pydub import AudioSegment
+import sys
+sys.path.append('/usr/bin/ffmpeg')
+
+def audioTrimmer(file_name,startMin,startSec,endMin,endSec):
+  files_path = 'mp3/'
```
Comment (Basanta78) [inline musicrs/util/audioTrimmer.py]: files path should be in parameter.. Utils should be as generic as possible. Its only task is to trim the audio the necessary information should be given by the service file which uses this util

File: musicrs/util/audioTrimmer.py [hunk 6]
```diff
@@ -0,0 +1,17 @@
+from pydub import AudioSegment
+import sys
+sys.path.append('/usr/bin/ffmpeg')
```
Comment (Basanta78) [inline musicrs/util/audioTrimmer.py]: Why are we changing the path here?

**Review by Basanta78 — APPROVED** [review]

**Pull request merged**